noxfile.py

```diff
@@ -47,6 +51,27 @@ def unit(session):
     )
 
 
+def check_protoc(session):
+    """Make sure the Protocol Compiler (protoc) is installed"""
+    session.install("packaging")
```

Comment:
I just realized this won't work for making sure `packaging` is available in the noxfile. 😅  Will modify